.github/workflows/triage_add_to_routed_project.yml

```diff
@@ -0,0 +1,39 @@
+name: Triage add to routed-to project
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-to-e2e:
+    if: github.event.label.name == 'routed-to-e2e'
```

Comment:
any way to update this handle (or is this coming work) dragging issue to column and adding this label & adding to e2e board?

Reply:
Unfortunately, there aren't any built-in project workflows for changing a fields value and Projects (beta) isn't a trigger event for GitHub Actions.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#project